tst/EnergyPlus/unit/OutputReportTabular.unit.cc

```diff
@@ -235,6 +235,29 @@ TEST( OutputReportTabularTest, GetUnitConversion )
 
 }
 
+TEST( OutputReportTabularTest, GetColumnUsingTabs )
+{
+	ShowMessage( "Begin Test: OutputReportTabularTest, GetColumnUsingTabs" );
+
+	std::string inString1 = " Col1 \t Col2 \t Col3 ";
+	EXPECT_EQ( " Col1 ", GetColumnUsingTabs( inString1, 1 ) );
+	EXPECT_EQ( " Col2 ", GetColumnUsingTabs( inString1, 2 ) );
+	EXPECT_EQ( " Col3 ", GetColumnUsingTabs( inString1, 3 ) );
+	EXPECT_EQ( "", GetColumnUsingTabs( inString1, 4 ) );
+
+	std::string inString2 = "Col1\tCol2\tCol3";
+	EXPECT_EQ( "Col1", GetColumnUsingTabs( inString2, 1 ) );
+	EXPECT_EQ( "Col2", GetColumnUsingTabs( inString2, 2 ) );
+	EXPECT_EQ( "Col3", GetColumnUsingTabs( inString2, 3 ) );
+	EXPECT_EQ( "", GetColumnUsingTabs( inString2, 4 ) );
+
+	std::string inString3 = "Col1\tCol2\tCol3\t";
+	EXPECT_EQ( "Col1", GetColumnUsingTabs( inString3, 1 ) );
+	EXPECT_EQ( "Col2", GetColumnUsingTabs( inString3, 2 ) );
+	EXPECT_EQ( "Col3", GetColumnUsingTabs( inString3, 3 ) );
+	EXPECT_EQ( "", GetColumnUsingTabs( inString3, 4 )  );
```

Comment:
@JasonGlazer As I scan through this unit test, I'm thinking it might help robustness if we add a couple more possibilities, unless you think they aren't needed.  My thoughts are:

 - `std::string inString4 = "";`
 - `std::string inString5 = " ";`
 - `std::string inString6 = "\t";`
 - `std::string inString7 = "\tCol1\tCol2\tCol3\t";`
 - `std::string inString3 = "Col1\t\tCol2\tCol3\t";`

Would you mind adding those cases in and making sure the function passes all those cases?  If so this should be good to go right in, and I'd really feel good we were exercises a broad spectrum of possible cases.